packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -567,6 +567,8 @@ class MemoryController extends DisposableController
     _memoryTracker.start();
 
     // Log Flutter extension events.
+    // Note: We do not need to listen to event history here because we do not
+    // have matching historical data about total memory usage.
```

Comment:
That's correct. Today we could start collecting memory from the beginning of time - if the app to profile was started with `--start-paused`. When DevTools is up:
1. flip to the memory page
2. flip to debugger
3. press GO

I'm wondering should we have a simpler way like a setting to "start collect memory statistics on DevTools attach"?  Doesn't effect anything today just a question if anyone has a suggestion.